cloudtrust-test-tools/pom.xml

```diff
@@ -13,6 +13,8 @@
         <maven.compiler.target>1.8</maven.compiler.target>
 
         <junit.version>4.12</junit.version>
+        <spotbugs.version>4.0.0</spotbugs.version>
```

Comment:
If already declared in cloudtrust-parent, we won't need this property here.
junit.version and maven.compiler.* can be moved too

Reply:
Indeed, moved to cloudtrust-parent pom.